From 8bebad2ab2e41bfb01edee8e6ee78fa2c897a8bd Mon Sep 17 00:00:00 2001
From: gord chung <gord@live.ca>
Date: Mon, 23 Oct 2017 16:46:24 +0000
Subject: [PATCH 76/91] fix radosgw meter name

meter name in entrypoint should match actual meter name so it can
probably be enabled/filtered in polling/pipeline

Change-Id: I8ffbfc5ff65c505734ccc6c5fd4270ecf46d16e8
Closes-Bug: #1726458
---
 releasenotes/notes/fix-radosgw-name-6de6899ddcd7e06d.yaml | 15 +++++++++++++++
 setup.cfg                                                 |  7 +++++++
 2 files changed, 22 insertions(+)
 create mode 100644 releasenotes/notes/fix-radosgw-name-6de6899ddcd7e06d.yaml

diff --git a/releasenotes/notes/fix-radosgw-name-6de6899ddcd7e06d.yaml b/releasenotes/notes/fix-radosgw-name-6de6899ddcd7e06d.yaml
new file mode 100644
index 0000000..f143c51
--- /dev/null
+++ b/releasenotes/notes/fix-radosgw-name-6de6899ddcd7e06d.yaml
@@ -0,0 +1,15 @@
+---
+upgrade:
+  - |
+    Use `radosgw.*` to enable/disable radosgw meters explicitly rather than
+    `rgw.*`
+deprecations:
+  - |
+    Previously, to enable/disable radosgw.* meters, you must define entry_point
+    name rather than meter name. This is corrected so you do not need to be
+    aware of entry_point naming. Use `radosgw.*` to enable/disable radosgw
+    meters explicitly rather than `rgw.*`. `rgw.*` support is deprecated
+    and will be removed in Rocky.
+fixes:
+  - |
+    Fix ability to enable/disable radosgw.* meters explicitly
diff --git a/setup.cfg b/setup.cfg
index 58f74fe..37665b2 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -157,12 +157,19 @@ ceilometer.poll.central =
     port.transmit.bytes = ceilometer.network.statistics.port_v2:PortPollsterTransmitBytes
     port.receive.drops = ceilometer.network.statistics.port_v2:PortPollsterReceiveDrops
     port.receive.errors = ceilometer.network.statistics.port_v2:PortPollsterReceiveErrors
+    # rgw.* are incorrect and deprecated to be removed in Rocky
     rgw.containers.objects = ceilometer.objectstore.rgw:ContainersObjectsPollster
     rgw.containers.objects.size = ceilometer.objectstore.rgw:ContainersSizePollster
     rgw.objects = ceilometer.objectstore.rgw:ObjectsPollster
     rgw.objects.size = ceilometer.objectstore.rgw:ObjectsSizePollster
     rgw.objects.containers = ceilometer.objectstore.rgw:ObjectsContainersPollster
     rgw.usage = ceilometer.objectstore.rgw:UsagePollster
+    radosgw.containers.objects = ceilometer.objectstore.rgw:ContainersObjectsPollster
+    radosgw.containers.objects.size = ceilometer.objectstore.rgw:ContainersSizePollster
+    radosgw.objects = ceilometer.objectstore.rgw:ObjectsPollster
+    radosgw.objects.size = ceilometer.objectstore.rgw:ObjectsSizePollster
+    radosgw.objects.containers = ceilometer.objectstore.rgw:ObjectsContainersPollster
+    radosgw.usage = ceilometer.objectstore.rgw:UsagePollster
     switch.port = ceilometer.network.statistics.port:PortPollster
     switch.port.uptime = ceilometer.network.statistics.port:PortPollsterUptime
     switch.port.receive.packets = ceilometer.network.statistics.port:PortPollsterReceivePackets
-- 
2.7.4

